Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.cs

```diff
@@ -0,0 +1,126 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents a control that displays a header that has a collapsible window that displays content.
+    /// </summary>
+    [TemplateVisualState(Name = "ExpandedDown", GroupName = "CommonStates")]
+    [TemplateVisualState(Name = "CollapsedDown", GroupName = "CommonStates")]
+    [TemplateVisualState(Name = "ExpandedUp", GroupName = "CommonStates")]
+    [TemplateVisualState(Name = "CollapsedUp", GroupName = "CommonStates")]
+    [TemplatePart(Name = "ExpanderButton", Type = typeof(Button))]
```

Comment:
My only concern with using ButtonBase or ToggleButton is wouldn't that make the entire area clickable including the expanded content?  I suppose we could intercept the click event and cancel it?